crates/lunatic-distributed-api/Cargo.toml

```diff
@@ -9,6 +9,7 @@ license = "Apache-2.0/MIT"
 
 [dependencies]
 lunatic-common-api = { workspace = true }
+lunatic-control-axum = { workspace = true }
```

Comment:
No, no, if you need some types that are used both in control axum and here, now you have lunatic-control which should keep types in-common.

Reply:
The reason for this dependency is so that I can use `ApiError`. Should I move `ApiError` to lunatic-control? If so, I suspect that I will need to introduce from lunatic-control to axum, is that alright?

Reply:
I think you don't need them. ApiError is an internal thing, the external interface is HTTP. In the HTTP client you can just match code strings  and HTTP statues which are meaningful to handle. Otherwise, you just report error.